locales/it/index.yml

```
@@ -1016,7 +1016,9 @@ messages:
       addMemo:
         block1: Aggiungi un promemoria per il
         block2: alle
-      expiringAlert: Questo avviso scade oggi alle
+      expiringAlert:
+        block1: Questo avviso scade il giorno
```

If you need to have a 2 lines text you could refactor like
```suggestion
        block1: Questo avviso scade il giorno {{date}} \n alle ore {{time}}
```

---

Really what I would like to do is make `date` and `time` bold.